demo/examples/Modules/model_with_deps.sysl

```diff
@@ -6,6 +6,8 @@ import moredeps/moredeps
 import /tests/bananatree
 # To import a sysl file from external repo. Add // at the beginning of the path
 import //github.com/anz-bank/syslgen-examples/examples/todos
+# To import a sysl file from external repo with fixed version
+import //github.com/anz-bank/syslgen-examples/examples/todos@v0.2.0
```

Comment:
```suggestion
import //github.com/anz-bank/syslgen-examples/examples/todos@0.0.1
```
v0.2.0 doesn't exist, so it won't work if someone tries it, but 0.0.1 does https://github.com/anz-bank/syslgen-examples/tree/0.0.1/examples